src/table/components/footer/PaginationContent.tsx

```diff
@@ -124,11 +125,12 @@ const PaginationContent = ({
   ) => {
     const label = translator.get(`SNTable.Pagination.${name}`);
     const id = `${name}-dropdown`;
-    const labelId = `${id}-label`;
+    const labelId = `${id}-label-${chartId}`;
```

Comment:
It is possible to render the same chart twice on a page. Not sure if we need to cover that case...

Reply:
If you have a master viz or multiple instances in a mashup?